lib/takeaway.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_literial_string: true
+
+require 'twilio_client'
+
+class TakeAway
+  attr_reader :basket
+
+  def initialize(output)
+    @basket = []
+    @output = output
+    @menu = {
+      'spring roll' => 0.99,
+      'char sui bun' => 3.99,
+      'pork dumpling' => 2.99,
+      'peking duck' => 7.99,
+      'fu-king fried rice' => 5.99
+    }
+  end
+
+  def read_menu
+    menu.each { |key, value| output.puts "#{key}: £#{value}" }
+  end
+
+  def add_dish(dish)
+    check_for_string(dish)
+    return print_error_message unless menu.keys.any?(dish.downcase)
+
+    @basket << menu.select { |key| key == dish.downcase }
+  end
+
+  def check_order
```

Comment:
Maybe you could separate methods related to the order into their own class, away from menu related methods.

Reply:
That sound like a good idea. I think I need to ask you more about this so I get it right.